contents/blog/growth-loops.md

```diff
@@ -0,0 +1,121 @@
+---
+date: 2022-10-19
+title: "How to choose and measure growth loops"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/posthog-marketing/marketing-hog.jpg
+featuredImageType: full
+categories: ["Product analytics", "Guides"]
+---
+
+A growth loop is a series of actions that amplify each other to fuel perpetual growth. New inputs repeatedly power the series to exponentially grow the whole. Companies use growth loops to align their product and distribution to have the greatest impact.
+
+Growth loops are an alternative way to thinking than [funnels](/blog/aarrr-pirate-funnel). While funnels focus on a series of steps from Acquisition to Referral, growth loops focus on a smaller number of steps done repeatedly.
+
+## Why you should care about growth loops
+
+Growth loops are important because they compound. Ideally, the growth in your key metrics exponentially increases, while the average effort or investment decreases. For many companies, this key metric is users or revenue which helps create a successful company. 
+
+Nearly every successful company utilizes growth loops in some way, consciously or not. For example, the newsletter platform Substack allows writers to easily share and monetize their work. As writers share their work (for their own benefit), it gets the attention of other writers, who see the benefit of the platform, join, and continue the loop.
+
+Growth loops can be a useful way of aligning multiple areas. For a growth loop to work, customer-facing teams must align with engineering and product teams. Each team benefits from the work the others are doing. 
+
+In our Substack example, the marketing and business development teams have to attract large writers (who often required upfront payments). The product and engineering teams then have to deliver a great writing and subscription experience. Without both, the growth loop doesn't work. When the company is aligned, however, it can cause exponential growth through compounding benefits. For Substack, that means bigger writers, better experiences, more readers, and ultimately, more revenue.
```

Comment:
```suggestion
In our Substack example, the marketing and business development teams have to attract popular writers (who often require upfront payments). The product and engineering teams then have to deliver a great writing and subscription experience. Without both, the growth loop doesn't work. When the company is aligned, however, it can cause exponential growth through compounding benefits. For Substack, that means bigger writers, better experiences, more readers, and ultimately, more revenue.
```

Reply:
Ok, asking Lottie.